src/lib/lldpctl.h

```diff
@@ -714,6 +714,7 @@ typedef enum {
 	lldpctl_k_config_ifdescr_update, /**< `(I,WO)` Enable or disable setting interface description */
 	lldpctl_k_config_iface_promisc,  /**< `(I,WO)` Enable or disable promiscuous mode on interfaces */
 	lldpctl_k_config_chassis_cap_advertise, /**< `(I,WO)` Enable or disable chassis capabilities advertisement */
+	lldpctl_k_config_chassis_cap_override, /**< `(I,WO)` Override chassis capabilities */
```

Comment:
You need to move it to the end of the block. Otherwise, it would break existing clients.